runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -172,6 +176,10 @@ public static void main(String[] args) throws Exception
 		final OptionParser parser = new OptionParser();
 		parser.accepts("developer-mode", "Enable developer tools");
 		parser.accepts("debug", "Show extra debugging output");
+		parser.accepts("sessionfile", "Use a specified session file")
+			.withRequiredArg().defaultsTo(DEFAULT_SESSION_FILE);
+		parser.accepts("config", "Use a specified config file")
```

Comment:
Did you try `ofType(File.class);`? Does that work?

Reply:
I wanted the input as a string so I can change the directory to the RL directory. The most common usecase will be args in the launcher with users that will not understand that the file is going into the cwd. My current implementation also lets users use the cwd or an absolute path if they want.